app/gui/src/project-view/components/GraphEditor/widgets/WidgetTableEditor/__tests__/tableInputArgument.test.ts

```diff
@@ -174,16 +174,17 @@ test.each([
 )
 
 test.each([
-  'Table.new 14',
-  'Table.new array1',
-  "Table.new ['a', [123]]",
-  "Table.new [['a', [123]], ['b', [124], []]]",
-  "Table.new [['a', [123]], ['a'.repeat 170, [123]]]",
-  "Table.new [['a', [1, 2, 3, 3 + 1]]]",
+  'Table.input 14',
+  'Table.input array1',
+  "Table.input ['a', [123]]",
+  "Table.input [['a', [123]]]",
```

Comment:
Besides this line, all other cases should have `'123'`  - otherwise we could get false positive (for example, we want to test that we won't match against a column name which is a function call (line 182), but even if the code has a bug and we do match, the test still passes because of number presence.